.github/workflows/phar-creation.yml

```diff
@@ -0,0 +1,66 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Building Require-Checker phar"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - "master"
```

Comment:
There won't be a `master` branch anymore very soon, just be aware of that.

Can we drop this?

Reply:
Sure thing